code/modules/surgery/organs/organ_icon.dm

```diff
@@ -171,10 +171,10 @@ var/global/list/limb_icon_cache = list()
 			icon_file = 'icons/mob/human_races/robotic.dmi'
 		else
 			if(status & ORGAN_MUTATED)
-				icon_file = species.deform
+				icon_file = owner ? owner.deform : species.deform
 			else
 				// Congratulations, you are normal
-				icon_file = species.icobase
+				icon_file = owner ? owner.icobase : species.icobase
```

Comment:
Yeah it's busted, back to work! I'm going to have it per-limb, set up in external organs New() where if there's a holder it'll get the holder's icobase and if there's not a holder it'll get the species icobase of src (needs a default) and I'll be changing these lines so they equal src's deform and icobase.